ProfileView/src/[sandbox]/sandbox.ts

```diff
@@ -92,7 +92,7 @@ export class Sandbox extends LitElement {
             <!-- profile-read-token="so=demoassure&sn=sandbox&ss=profile&sp=r&se=2022-06-17T23:47:34.409Z&sk=sandbox&sig=61BLCJ5+vZtOOvut/7khUQyg0N9KlvbPrJWrYa9lf28=" -->
             <cjaas-profile-view-widget
               id="view"
-              customer="560021-Venki"
+              customer="30313-Alex"
               .template=${sampleTemplate}
               profile-write-token="so=demoassure&sn=sandbox&ss=profile&sp=w&se=2022-06-17T21:36:08.050Z&sk=sandbox&sig=gm%2FXQ%2Bjtu8uWPrUtpRfR6P4DHwrJV2CJokIH3BcgzdE%3D"
```

Comment:
We need to update these SAS tokens to the new ones we mad last week when they were rotated, right now the API calls fail